animations/test/empty_test.dart

```diff
@@ -1,9 +0,0 @@
-// Copyright 2019 The Flutter team. All rights reserved.
```

Comment:
Without at least one test file in the /test directory, the CI routine will error out.

This would be a great opportunity to add a real widget test, though. 😄